envs/ci/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: "3.8"
```

Comment:
Since we're using compose v2 in our CI pipelines, let's remove this version parameter, because it's deprecated in compose V2 (see [link](https://docs.docker.com/compose/compose-file/04-version-and-name/))

Reply:
Also I would suggest to add a `name` parameter to every compose file we have, to make containers/networks/volumes defined in this file to be unique across all other containers which may be run on developer's machine (for example if s/he develops more than one project).